src/components/Popup/Popup.tsx

```diff
@@ -139,23 +150,29 @@ export function Popup({
                     id={id}
                     role={role}
                 >
-                    {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
-                    <div
-                        onClick={onClick}
-                        onMouseEnter={onMouseEnter}
-                        onMouseLeave={onMouseLeave}
-                        className={b('content', contentClassName)}
-                        style={style}
-                    >
-                        {hasArrow && (
-                            <PopupArrow
-                                styles={styles.arrow}
-                                attributes={attributes.arrow}
-                                setArrowRef={setArrowRef}
-                            />
-                        )}
-                        {children}
-                    </div>
+                    <FocusTrap enabled={focusTrap && open} disableAutoFocus={!autoFocus}>
+                        {/* The event handlers should only be used to capture bubbled events */}
+                        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+                        <div
+                            onClick={onClick}
+                            onKeyDown={onKeyDown}
```

Comment:
Didn't notice this at first. Why we're listening keydown?

Reply:
The onClick event handler is currently used to execute an action within a popup when it's clicked. It returns true or false, which determines whether to close the popup. This behavior should also work for keyboard and screen reader users. I can't think of a specific use case for onClick at the moment, but in any case, if there's an action involved, it should be accessible to users who don't use a mouse.

Reply:
I think onClick here is by mistake. Popup should not handle clicks or keyboard events, it's utility component.

Reply:
I marked onClick as deprecated and removed onKeyDown